client/my-sites/plans/jetpack-plans/product-store/index.tsx

```diff
@@ -1,11 +1,22 @@
+import { useSelector } from 'react-redux';
 import StoreFooter from 'calypso/jetpack-connect/store-footer';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { JetpackFree } from './jetpack-free';
 import { UserLicensesDialog } from './user-licenses-dialog';
 import type { ProductStoreProps } from './types';
 
-const ProductStore: React.FC< ProductStoreProps > = ( { enableUserLicensesDialog } ) => {
+import './style.scss';
+
+const ProductStore: React.FC< ProductStoreProps > = ( {
+	enableUserLicensesDialog,
+	urlQueryArgs,
+} ) => {
+	const siteId = useSelector( getSelectedSiteId );
```

Comment:
Hi I notice that `siteId` is also in `jetpack-plans/selector.tsx`.  Should we also pass it here as prop?

Reply:
Nice thought James, but I didn't pass that because the previous component (`ProductGrid`) that is replaced by `ProductStore` didn't get that prop from the parent, so I decided to keep it similar.
Thank you.

Reply:
Yup sounds good to me 👍